.gitignore

```
@@ -1,4 +1,7 @@
 .project
 .settings
 .classpath
-/bin
\ No newline at end of file
+/bin/
+/.idea/
```

Para ignorar la carpeta .idea y todo su contenido usar flags
```suggestion
.idea/*
```